2020q3/crossbuilding.md

```diff
@@ -0,0 +1,49 @@
+## Building FreeBSD on non-FreeBSD hosts ##
+
+Link:	 [Wiki](https://wiki.freebsd.org/BuildingOnNonFreeBSD)  
+
+Contact: Alex Richardson <arichardson@freebsd.org>  
+
+Until recently FreeBSD could only be built on a FreeBSD host.
+However, most free CI tools only allow building on Linux or macOS and therefore
```

Comment:
Cirrus-CI has supported FreeBSD for quite some time now and Travis has also added support I believe, so IMO we should soften this statement slightly. Maybe "many popular hosted CI tools" or something similar?